### DISCUSSION TITLE
FlorisBoard Docs

### DISCUSSION THREAD

**Discussion by madalv** [florisboard/florisboard#1952] [General]
Hi, I just wanted to ask if there is documentation available or planned for FlorisBoard. I'd like to try to contribute to the app sometime in the near future after I understand the codebase better & some documentation would be pretty handy. Alternatively, is there any place where people could ask for help/understand the codebase better?

**Comment by patrickgold** [comment]
No, not really atm as a lot of the code base is still voilatile, although some interfaces and classes have extensive KDoc comments attached to them or general comments in the code.

As for asking for help: you can do it here on GH or in the Matrix group chat (see README).
